model/src/main/kotlin/config/S3FileStorageConfiguration.kt

```diff
@@ -38,6 +38,9 @@ data class S3FileStorageConfiguration(
     /** Custom endpoint to perform AWS API Requests */
     val customEndpoint: String? = null,
 
+    /** Whether to enable path style access or not. Required for many non-AWS S3 providers. Defaults to false. */
+    val pathStyleAccess: Boolean = false,
```

Comment:
Commit message nit: This now again only exposes *one* more client option, namely `pathStyleAccess`. Maybe the commit message should be adjusted accordingly to be more specific.